lib/plugins/aws/package/compile/events/apiGateway/lib/cors.test.js

```diff
@@ -157,7 +157,7 @@ describe('#compileCors()', () => {
         awsCompileApigEvents.serverless.service.provider.compiledCloudFormationTemplate.Resources
           .ApiGatewayMethodUsersUpdateOptions.Properties.Integration.IntegrationResponses[0]
           .ResponseParameters['method.response.header.Access-Control-Allow-Credentials']
-      ).to.equal("'false'");
+      ).to.equal(undefined);
```

Comment:
Let's change to `to.not.exist`

Reply:
Or what about - `to.be.undefined`? (As recommended in the chai over `.to.not.exist`: @see https://www.chaijs.com/api/bdd/)

```
expect(1).to.equal(1); // Recommended
expect(1).to.exist; // Not recommended

expect(0).to.equal(0); // Recommended
expect(0).to.exist; // Not recommended
```

Add .not earlier in the chain to negate .exist.

```
expect(null).to.be.null; // Recommended
expect(null).to.not.exist; // Not recommended

expect(undefined).to.be.undefined; // Recommended
expect(undefined).to.not.exist; // Not recommended
```

Reply:
I assumed that `.to.not.exists` checks whether property is defined, while it checks for `null` or `undefined` value, which indeed is not what the name advertises.

In that case, yes let's use `to.be.undefined`

Reply:
Totally agree, the naming indicates a missing of a property. Change is on the way. Thanks for your review.